core/src/test/java/hudson/util/ConsistentHashTest.java

```diff
@@ -135,7 +141,6 @@ public void testSpeed() {
         for (int j=0; j<10; j++) {
             ConsistentHash<String> b = new ConsistentHash<String>();
             b.addAll(data);
-//            System.out.println(Iterables.toString(b.list("x")));
```

Comment:
Can leave such things in; was probably useful at some point when developing the test.